config/root/clusterworkspacetype-home.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: tenancy.kcp.dev/v1alpha1
+kind: ClusterWorkspaceType
+metadata:
+  name: home
+spec:
+  initializer: true
+  defaultChildWorkspaceType: Universal
+  allowedChildWorkspaceTypes:
+  - Any
+  allowedParentWorkspaceTypes:
+  - Homebucket
```

Comment:
nit: HomeBucket

Reply:
I cannot do that. Upper case letters inside the type name are not possible (due to implementation of `ClusterWorkspaceType`s)

Reply:
I see `[A-Z][a-zA-Z0-9]+` in the OpenAPI.

Reply:
Not sure we enforce that only the first letter is upper-case. Do we? cc @stevekuznetsov